src/DynamoCoreWpf/ViewModels/PackageManager/PackagePathViewModel.cs

```diff
@@ -184,12 +199,35 @@ private void RemovePathAt(int index)
 
         private void CommitChanges(object param)
         {
-            setting.CustomPackageFolders = new List<string>(RootLocations);
+            setting.CustomPackageFolders = CommitRootLocations();
             if (this.packageLoader != null)
             {
                 this.packageLoader.LoadCustomNodesAndPackages(loadPackageParams, customNodeManager);
             }
         }
 
+        private void InitializeRootLocations()
+        {
+            RootLocations = new ObservableCollection<string>(setting.CustomPackageFolders);
+            var index = RootLocations.IndexOf(StandardLibraryToken);
+
+            if (index != -1)
+            {
+                RootLocations[index] = Resources.PackagePathViewModel_Standard_Library;
+            }
+        }
+
+        private List<string> CommitRootLocations()
+        {
+            var rootLocations = new List<string>(RootLocations);
+            var index = rootLocations.IndexOf(Resources.PackagePathViewModel_Standard_Library);
+
+            if (index != -1)
+            {
+                rootLocations[index] = StandardLibraryToken;
```

Comment:
What does it mean for the list of root locations to contain the `StandardLibraryToken` as opposed to `Resources.PackagePathViewModel_Standard_Library` string?

Reply:
The token is what we store in the settings file. It stays the same regardless of what language you run Dynamo in. The resource string is what we display and will say Standard Library in the current language.